### DISCUSSION TITLE
Welcome to CoreFreq Discussions!

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#233] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other users of **_CoreFreq_**. 

We hope that you:
  * Ask questions you’re wondering about _CoreFreq_.
  * Share ideas.
  * Share screenshots 💪.

**Comment by jowa2021** [comment]
Hi, sorry really busy lately. Anyway, I noticed the email about new CPU data needed, I can help I am sure, I would just need to know what data is needed, and then just a bit of time after that. Also, the old error i posted previously somewhere didnt go away, but has not come back either, as I dont even remember what system or conditions triggered that... so I guess forget it for now :\ I'd like to help out as much as possible, but please keep in mind I don't always check here, but for sure will get around to doing so ! (hopefully more often as well) :) Ok, have a good one!

**Reply by cyring** [reply]
Hello,

I would suggest to check if your processor is within the already published ones and results are coherent.
See the [Wiki/CPU Support](https://github.com/cyring/CoreFreq/wiki/CPU-support) where Contributors are supplying their data.

For sure, if _CoreFreq_ is crashing right away, please fill an issue, titled with the processor brand name.
I will be interested by the part of `dmesg` where `corefreqk.ko` driver is involved, showing the kernel trapped registers and the functions callflow.
